README.md

```diff
@@ -1,2 +1,130 @@
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+> This is my first Microverse Project. It contains every first step I have to take as a developer.
+
+**[Hello Microverse Project]** is a website that is used to test for different linter errors that could occur when building a HTML and CSS project.
+
+## 🛠 Built With <a name="HTML and CSS"></a>
+
+### Tech Stack <a name="Front end"></a>
+
+> This is a desk-top browser website.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Reset CSS]**
+- **[Changed the content color]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser that supports html5 and css3
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/chiedozie013/Hello-world.git
+```
+
+### Install
+
+Install this project with:
+
+- Clone the code into your machine and run test in an editor of your choice(E.g VScode).
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/chiedozie013)
+- Twitter: [@twitterhandle](https://twitter.com/edoziey)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/chiedozie013/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Contact form]**
+- [ ] **[Mobile version of the website]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/chiedozie013/Hello-world/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project or would love to tell me ways to improve it, contact me on LinkedIn or Twitter
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank Microverse community for this opportunity to learn new things.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly ensure that you add the `MIT` link in your readme under this section. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose.